sandbox-backend/src/main/java/com/exadel/sandbox/team5/dao/util/SearchCriteria.java

```diff
@@ -0,0 +1,30 @@
+package com.exadel.sandbox.team5.dao.util;
+
+import org.springframework.data.domain.PageRequest;
+
+import java.util.List;
+
+public class SearchCriteria {
+    private PageRequest pageRequest;
+    private List<String> properties;
```

Comment:
What about types Date and Number for filter parameters and how we will understand what the value for what field

Reply:
Done